dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -1,137 +1,339 @@
-            :input-class="{
-              'formkit-input': false,
-              'p-radiobutton:': true,
-            }"
-          />
-          <FormKit type="group" name="eligibilityKpis" label="Eligibility KPIs">
-            <template
-              v-for="fsType in ['CreditInstitution', 'InsuranceOrReinsurance', 'AssetManagement', 'InvestmentFirm']"
-              :key="fsType"
-            >
-              <div :name="fsType">
-                <FormKit type="group" :name="fsType">
-                  <h4>Eligibility KPIs ({{ humanizeString(fsType) }})</h4>
-                  <DataPointFormElement name="taxonomyEligibleActivity" label="Taxonomy Eligible Activity" />
-                  <DataPointFormElement name="taxonomyNonEligibleActivity" label="Taxonomy Non Eligible Activity" />
-                  <DataPointFormElement name="derivatives" label="Derivatives" />
-                  <DataPointFormElement name="banksAndIssuers" label="Banks and Issuers" />
-                  <DataPointFormElement name="investmentNonNfrd" label="Investment non Nfrd" />
-                </FormKit>
+            <div class="uploadFormSection grid">
+              <div class="col-3 p-3 topicLabel">
+                <h4 id="reportingPeriod" class="anchor title">Reporting Period</h4>
+              </div>
+              <div class="col-9 formFields uploaded-files">
+                <UploadFormHeader
+                  data-test="reportingPeriodLabel"
+                  :name="euTaxonomyKpiNameMappings.reportingPeriod"
+                  :explanation="euTaxonomyKpiInfoMappings.reportingPeriod"
+                />
+                <div class="md:col-6 col-12 p-0">
+                  <Calendar
+                    data-test="reportingPeriod"
+                    v-model="reportingPeriod"
+                    inputId="icon"
+                    :showIcon="true"
+                    view="year"
+                    dateFormat="yy"
+                  />
+                </div>
+
+                <FormKit type="hidden" v-model="reportingPeriodYear" name="reportingPeriod" />
+              </div>
+              <FormKit type="group" name="data" label="data">
+                <UploadReports
+                  ref="UploadReports"
+                  :filesToUpload="filesToUpload"
+                  :uploadFiles="uploadFiles"
+                  :euTaxonomyKpiNameMappings="euTaxonomyKpiNameMappings"
+                  :euTaxonomyKpiInfoMappings="euTaxonomyKpiInfoMappings"
+                  :maxFileSize="maxFileSize"
+                  :editMode="editMode"
+                  @selectedFiles="onSelectedFilesHandler"
+                  @removeReportFromFilesToUpload="removeReportFromFilesToUpload"
+                  @updateReportDateHandler="updateReportDateHandler"
+                />
+
+                <BasicInformationFields
+                  :euTaxonomyKpiNameMappings="euTaxonomyKpiNameMappings"
+                  :euTaxonomyKpiInfoMappings="euTaxonomyKpiInfoMappings"
```

Comment:
These two props are always the same in all current usages of `<BasicInformationFields>`.
So could we just import and use them inside of the component itself? Then we don't need these two props anymore.

Reply:
We could remove them if we assumed that the UploadReports and BasicInformationFields components would not be used with a different set of mapping variables

Reply:
I think for now we can assume that the `<BasicInformationFields>` are only used with Eu Taxonomy frameworks.
So at least there, we can remove these props and import the mappings directly.

For the `<UploadReports>` we can keep the props for now. This is more likely to be re-used in other frameworks.

Reply:
removed from <BasicInformationFields>